src/js/storage.js

```diff
@@ -326,25 +326,33 @@ const storage = {
             return;
           }
           let services = JSON.parse(res.data).data;
+          let servicesToSave = [];
+          let toReturn = {
+            services: new Array(),
```

Comment:
```suggestion
            services: [],
```
Não está errado usar `new Array()`, mas `[]` é mais amplamente e menor de escrever/ler.